universal-login-react/src/playground/App.tsx

```diff
@@ -272,7 +273,7 @@ export const App = () => {
                 render={({history}) => <EnterPassword
                   hideModal={() => history.push('/')}
                   walletService={walletService}
-                  onConfirm={(password) => console.log({password})}/>}
+                  onConfirm={async (password) => {console.log({password});}}/>}
```

Comment:
why this is necessary?

Reply:
The interface of EnterPassword component has changed - now it requires a promise function

Reply:
Does it work? 
```ts
async (password) => console.log(password);
```